documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,139 @@
+* Manually, using Kubernetes command-line interface, `kubectl`.
+* WLDF rules and policies; when the rule is met the Administration Server sends a REST call to the operator which calls the Kubernetes API to start a new pod/container/server.
+* We have developed and made open source the [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter) which exports WebLogic metrics to Prometheus and Grafana.  In Prometheus, you can set rules similar to WLDF and when these rules are met, a REST call is made to the operator which invokes a Kubernetes API to start a new pod. See to blog, [Announcing the New WebLogic Monitoring Exporter](https://blogs.oracle.com/weblogicserver/announcing-the-new-weblogic-monitoring-exporter-v2).
+
+
+**Q:** Container life cycle: How to properly spin up and gracefully shut down WLS in a container?
+
+**A:** The operator manages container/pod/WebLogic Server life cycle automatically; it uses the Node Manager (internally) and additional approaches  to do the following operations:
+* Entrypoint - start WebLogic Server.
+* Liveliness probe – check if the WebLogic Server is alive.
+* Readiness probe – query if the WebLogic Server is ready to receive requests.
+* Shutdown Hook – gracefully shut down WebLogic Server.
+
+These operations also can be done manually using the Kubernetes command-line interface, `kubectl`.
+
+***
+
+
+#### Patching and Upgrades
+
+**Q:** Patching: rolling upgrades, handling of one-off-patches and overlays, CPUs, and such.
+
+**A:** Patches are applied using OPatch and rolled out in the following fashion:
+* Upgrade or patch a WebLogic cluster in a Docker environment.
+* Apply PSUs for important security upgrades.
+* In Dockerfile, extend the image and apply patch using OPatch.
+* Operator starts rolling upgrade calling Kubernetes API.
+Starting the pods/containers/servers in a rolling fashion can be invoked manually, calling the Kubernetes API directly.  See the blog, [Patching WebLogic Server in a Kubernetes Environment](https://blogs.oracle.com/weblogicserver/patching-weblogic-server-in-a-kubernetes-environment).
+
+***
+
+#### Security
+
+**Q:** Certificates and CA provisioning and containers
+
+**A:** The operator will generate self-signed certificates based on the subject alternative names that you provide and will automatically configure the key stores, and such, for SSL. The alternative is that you can provide your own certificates that you get from a CA or that you create.
+
+***
+
+#### Diagnostics and Logging
+
+**Q:** Integration with Ecosystems: logging, monitoring (OS, JVM and application level), and such.
+
+**A:** WebLogic Server logs are persisted to an external volume.  We are working on a project to integrate WebLogic Server logs with the Elastic Stack.  
```

Comment:
are persisted --> are optionally persisted

Reply:
Or in more detail:  WebLogic Server stdout logging is echoed to their pod logs by default, and WebLogic Server file logs are optionally persisted to an external volume.

Reply:
Also, I think there's an Elastic stack sample now that can be linked here?

Reply:
Got it.